src/HString.php

```diff
@@ -232,10 +232,25 @@ public function toString()
     }
 
     /**
+     * Converts a string into an array. Assumes a delimiter of " ".
```

Comment:
Does this method still assume delimiter of `" "` ? I think it was changed to return an array of characters.